### PR TITLE
Mozambique

### PR DESCRIPTION
**Pull request by mr-shovel**
added Mozambique calendar
added tests
updated readme
updated changelog (did not know what to fill in with the (#) part

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -172,6 +172,7 @@ Africa
 * Madagascar
 * São Tomé
 * South Africa
+* Mozambique
```
Comment (brunobord) [inline README.rst:175]: please move "Mozambique" in the list so it's going to keep the alphabetical order.

**Review by brunobord — CHANGES_REQUESTED** [review]
Overall, this contribution is very good, tests and all!
There are a couple of items to make it pixel perfect. If you need some help, I'll surely be available.

I'm glad that a new African country is about to be added to Workalendar!

**Comment by mr-shovel** [comment]
Fixed your comments

**Comment by ewjoachim** [comment]
Hey :)

I think you have updated the **project**'s gitignore file.
What @brunobord suggested was to setup your **own** personal gitignore file, which would allow not needing every project you work on to make a special case for people with this IDE.

Here's the command you can run:
```console
$ echo .vscode > ~/.gitignoreglobal
$ git config --global core.excludesfile ~/.gitignoreglobal
```

**Comment by mr-shovel** [comment]
@ewjoachim @brunobord  Thanks for the remarks, learned a thing again today! Anything else needs resolving?

**Comment by ewjoachim** [comment]
I'm still seeing a diff on the .gitignore file. Is it intended?

(Thanks a lot for the contribution by the way)

**Comment by mr-shovel** [comment]
You're welcome:) I've removed the .vscode line from the gitignore. the .vscode folder should still be excluded.

**Review by ewjoachim — APPROVED** [review]

**Review by brunobord — APPROVED** [review]
:rocket: :mozambique:

**Comment by brunobord** [comment]
Mozambique calendar is now available through PyPI:

* https://pypi.org/project/workalendar/11.0.0/
* https://github.com/peopledoc/workalendar/releases/tag/11.0.0

Happy upgrading!

**Pull request merged**